npm/webpack-dev-server/src/makeWebpackConfig.ts

```diff
@@ -94,6 +92,42 @@ export function makeWebpackConfig (
     devServerEvents,
   } = config.devServerConfig
 
+  let configFile: string | undefined = undefined
+
+  if (!userWebpackConfig && !frameworkWebpackConfig) {
+    debug('Not user or framework webpack config received. Trying to automatically source it')
+
+    const { default: findUp } = await importModule('find-up')
+
+    configFile = await findUp(configFiles, { cwd: projectRoot } as { cwd: string })
+
+    if (configFile) {
+      debug('found webpack config %s', configFile)
+      const sourcedConfig = configFile.endsWith('mjs') ? await import(configFile) : require(configFile)
+
+      debug('config contains %o', userWebpackConfig)
```

Comment:
This will always be empty unless the debug is moved down a few lines
```suggestion
      debug('config contains %o', sourcedConfig)
```